### PR TITLE
[Outlook] (APIs) Clarify supported requirement sets

### PR DESCRIPTION
**Pull request by samantharamon**
- Clarifies platform and requirement set support for `Office.context.mailbox.item.addFileAttachmentAsync`.
- Clarifies supported scenarios and requirement sets for `Office.context.mailbox.item.getSharedPropertiesAsync`.

### CODE REVIEW

File: docs/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office.context.mailbox.item - preview requirement set
 description: Outlook Mailbox API preview requirement set version of the Item object model.
-ms.date: 07/18/2024
+ms.date: 11/07/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item.md:4]: make sure to update ms.date - here and other instances

```suggestion
ms.date: 11/19/2024
```

File: docs/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item.md [hunk 2]
```diff
@@ -110,8 +110,8 @@ ms.localizationpriority: medium
...
-| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-preview&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
+| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item.md:114]: hmm, "new Outlook" doesn't make it clear this is on Windows - here and other instances

Comment (samantharamon) [reply docs/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item.md:114]: Thanks for catching this. Yes, this should be "new Windows".

File: docs/requirement-sets/outlook/requirement-set-1.1/outlook-requirement-set-1.1.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Outlook add-in API requirement set 1.1
 description: Features and APIs that were introduced for Outlook add-ins and the Office JavaScript APIs as part of Mailbox API 1.1.
-ms.date: 02/02/2024
+ms.date: 11/07/2024
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.1/outlook-requirement-set-1.1.md:4]:
```suggestion
ms.date: 11/19/2024
```

File: docs/requirement-sets/outlook/requirement-set-1.10/office.context.mailbox.item.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office.context.mailbox.item - requirement set 1.10
 description: Outlook Mailbox API requirement set 1.10 version of the Item object model.
-ms.date: 07/18/2024
+ms.date: 11/07/2024
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.10/office.context.mailbox.item.md:4]:
```suggestion
ms.date: 11/19/2024
```

File: docs/requirement-sets/outlook/requirement-set-1.11/office.context.mailbox.item.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office.context.mailbox.item - requirement set 1.11
 description: Outlook Mailbox API requirement set 1.11 version of the Item object model.
-ms.date: 07/18/2024
+ms.date: 11/07/2024
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.11/office.context.mailbox.item.md:4]:
```suggestion
ms.date: 11/19/2024
```

File: docs/requirement-sets/outlook/requirement-set-1.12/office.context.mailbox.item.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office.context.mailbox.item - requirement set 1.12
 description: Outlook Mailbox API requirement set 1.12 version of the Item object model.
-ms.date: 07/18/2024
+ms.date: 11/07/2024
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.12/office.context.mailbox.item.md:4]:
```suggestion
ms.date: 11/19/2024
```

File: docs/requirement-sets/outlook/requirement-set-1.13/office.context.mailbox.item.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office.context.mailbox.item - requirement set 1.13
 description: Outlook Mailbox API requirement set 1.13 version of the Item object model.
-ms.date: 07/18/2024
+ms.date: 11/07/2024
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.13/office.context.mailbox.item.md:4]:
```suggestion
ms.date: 11/19/2024
```

File: docs/requirement-sets/outlook/requirement-set-1.14/office.context.mailbox.item.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office.context.mailbox.item - requirement set 1.14
 description: Outlook Mailbox API requirement set 1.14 version of the Item object model.
-ms.date: 07/18/2024
+ms.date: 11/07/2024
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.14/office.context.mailbox.item.md:4]:
```suggestion
ms.date: 11/19/2024
```

File: docs/requirement-sets/outlook/requirement-set-1.8/office.context.mailbox.item.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office.context.mailbox.item - requirement set 1.8
 description: Outlook Mailbox API requirement set 1.8 version of the Item object model.
-ms.date: 07/18/2024
+ms.date: 11/07/2024
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.8/office.context.mailbox.item.md:4]:
```suggestion
ms.date: 11/19/2024
```

File: docs/requirement-sets/outlook/requirement-set-1.8/outlook-requirement-set-1.8.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Outlook add-in API requirement set 1.8
 description: Requirement set 1.8 for Outlook add-in API.
-ms.date: 09/15/2022
+ms.date: 11/07/2024
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.8/outlook-requirement-set-1.8.md:4]:
```suggestion
ms.date: 11/19/2024
```

File: docs/requirement-sets/outlook/requirement-set-1.9/office.context.mailbox.item.md [hunk 11]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office.context.mailbox.item - requirement set 1.9
 description: Outlook Mailbox API requirement set 1.9 version of the Item object model.
-ms.date: 07/18/2024
+ms.date: 11/07/2024
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.9/office.context.mailbox.item.md:4]:
```suggestion
ms.date: 11/19/2024
```

File: docs/requirement-sets/outlook/requirement-set-1.9/office.context.mailbox.item.md [hunk 12]
```diff
@@ -99,8 +99,8 @@ ms.localizationpriority: medium
...
-| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.9&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.9&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
+| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.9&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.9/office.context.mailbox.item.md:103]:
```suggestion
| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.9&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.9&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
```

File: docs/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item.md (hunk 2 above)
Comment (samantharamon) [inline docs/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item.md:114]:
```suggestion
| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-preview&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
```

File: docs/requirement-sets/outlook/requirement-set-1.10/office.context.mailbox.item.md [hunk 13]
```diff
@@ -99,8 +99,8 @@ ms.localizationpriority: medium
...
-| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.10&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.10&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
+| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.10&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.10/office.context.mailbox.item.md:103]:
```suggestion
| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.10&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.10&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
```

File: docs/requirement-sets/outlook/requirement-set-1.11/office.context.mailbox.item.md [hunk 14]
```diff
@@ -101,8 +101,8 @@ ms.localizationpriority: medium
...
-| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.11&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.11&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
+| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.11&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.11/office.context.mailbox.item.md:105]:
```suggestion
| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.11&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.11&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
```

File: docs/requirement-sets/outlook/requirement-set-1.12/office.context.mailbox.item.md [hunk 15]
```diff
@@ -101,8 +101,8 @@ ms.localizationpriority: medium
...
-| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.12&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.12&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
+| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.12&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.12/office.context.mailbox.item.md:105]:
```suggestion
| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.12&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.12&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
```

File: docs/requirement-sets/outlook/requirement-set-1.13/office.context.mailbox.item.md [hunk 16]
```diff
@@ -104,8 +104,8 @@ ms.localizationpriority: medium
...
-| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.13&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.13&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
+| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.13&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.13/office.context.mailbox.item.md:108]:
```suggestion
| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.13&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.13&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
```

File: docs/requirement-sets/outlook/requirement-set-1.14/office.context.mailbox.item.md [hunk 17]
```diff
@@ -107,8 +107,8 @@ ms.localizationpriority: medium
...
-| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.14&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.14&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
+| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.14&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.14/office.context.mailbox.item.md:111]:
```suggestion
| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.14&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.14&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
```

File: docs/requirement-sets/outlook/requirement-set-1.8/office.context.mailbox.item.md [hunk 18]
```diff
@@ -99,8 +99,8 @@ ms.localizationpriority: medium
...
-| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.8&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.8&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
+| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.8&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
```
Comment (samantharamon) [inline docs/requirement-sets/outlook/requirement-set-1.8/office.context.mailbox.item.md:103]:
```suggestion
| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-1.8&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-1.8&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Windows) |
```

**Pull request merged**